geode-docs/developing/geode_for_redis.html.md.erb

```diff
@@ -49,20 +49,20 @@ Replace `<serverName>` with the name of your server.
 
 Replace `<locatorPort>` with your locator port.
 
-Replace `<compatibleWithRedisPort>` with the port that the <%=vars.product_name%> server
+Replace `<geodeForRedisPort>` with the port that the <%=vars.product_name%> server
  listens on for Redis commands. The typical port used with a Geode for Redis cluster is 6379.
 
-Replace `<compatibleWithRedisBindAddress>` with the address of the server host.
+Replace `<geodeForRedisBindAddress>` with the address of the server host.
 
 Replace `<compatibleWithWithRedisPassword>` with the password clients use to authenticate.
```

Comment:
This also needs to be updated. Easy to miss since it's a typo.

Reply:
Good catch!